cypress/e2e/admin_spec/actionItem.cy.ts

```
@@ -1,13 +1,152 @@
 import { AdminDashboardPage } from '../../pageObjects/AdminPortal/AdminDashboard';
 import { ActionItemPage } from '../../pageObjects/AdminPortal/ActionItemPage';
 
+/**
+ * Retrieve the auth token from localStorage.
+ * @returns Cypress.Chainable<string | null> token value or null if missing/invalid.
```

ESLint

[error] tsdoc-html-tag-missing-greater-than: The HTML tag has invalid syntax: Expecting an attribute or ">" or "/>"

(tsdoc/syntax)